### DISCUSSION TITLE
Registration

### DISCUSSION THREAD

**Discussion by catrin-eden** [IBAHCM/RPiR#169] [Archive]
Please can I be added to the organisation?

**Comment by richardreeve** [comment]
Hi Catrin, welcome. You're very late arriving on the course though. Do you really think you have time to catch up?

**Reply by catrin-eden** [reply]
Hi Richard, I am not officially enrolled on the course as I'm on the MRes so just utilising the material in my own time. I requested to join more so I could be notified of announcements. I hope that's ok?

**Reply by richardreeve** [reply]
Sure, no problem. I've sent you an invitation - you should be able to accept the invitation from your email or (hopefully!) [here](https://github.com/IBAHCM). Let me know if you have any problems.
